web/client/product/assets/css/viewer.css

```diff
@@ -435,6 +444,8 @@ html, body, #container, .fill {
 
 .modal-body {
     background-color: white;
+    height: calc(100vh - 190px);
```

Comment:
This makes the dialog height bigger than the original dialog.

![image](https://cloud.githubusercontent.com/assets/2770528/19510101/088c3ec2-95e2-11e6-9763-736a6c8bd2a2.png)